packages/angular/projects/clarity-adoption/src/app/pages/checkbox/checkbox.page.html

```diff
@@ -0,0 +1,21 @@
+<h1>Checkbox</h1>
+
+<h2>4.0.0</h2>
+
+<h3>Eslint plugin</h3>
+<p>
+  There is <b>eslint plugin</b> that could help with the adoption of Core Alerts. By locating and pinpoint the places
```

Comment:
`There is an...`

`...with the adoption of Core Checkboxes.`

`By locating and pinpointing...`